src/components/Dialog/ReportDialog.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <div>
+    <Dialog
+      v-if="iVisible"
+      :destroy-on-close="true"
+      :show-cancel="false"
+      :show-confirm="false"
+      :title="$tc('Report')"
+      :visible.sync="iVisible"
+      top="35vh"
+      width="80%"
+      @close="loading=true"
+    >
+      <span v-if="loading" v-loading="loading" class="loading" />
+      <iframe title="dialog" :src="url" style="border: none;" @load="onIframeLoad" />
+    </Dialog>
+  </div>
+</template>
+
+<script>
+import Dialog from '@/components/Dialog/index.vue'
+
+export default {
+  name: 'ReportDialog',
+  components: {
+    Dialog
+  },
+  props: {
+    visible: {
+      type: Boolean,
+      default: false
+    },
+    url: {
+      type: String,
+      default: ''
+    }
+  },
+  data() {
+    return {
+      loading: true
+    }
+  },
+  computed: {
+    iVisible: {
+      get() {
+        return this.visible
+      },
+      set(val) {
+        this.$emit('update:visible', val)
+      }
+    }
+  },
+  mounted() {
+  },
+  beforeMount() {
+  },
+  methods: {
+    onIframeLoad() {
+      this.loading = false
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+iframe {
+  width: 100%;
+  height: 500px;
+}
+
+.loading {
+  margin-top: 20px;
+  display: block;
+}
+</style>
```

Comment:
There is no major issue or improvement to suggest given the content you provided. The code appears correct with minimal differences across several files and versions of Vue.js and its related libraries. For reference, I recommend keeping an updated version for optimal performance and security; however, there are no evident issues in the snippet.